css/modules.css

```diff
@@ -0,0 +1,77 @@
+.main-nav ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+h1 {
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  font-size: 15vw;
+}
+
+h2 {
+  display: flex;
+  font-size: 8vw;
+}
+
+#img {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+img {
+  height: 400px;
+  width: 600px;
+  border: 1px solid black;
+  border-radius: 5vw;
+}
+
+
+
+li {
+  font-size: 5vw;
+}
+
+@media only screen and (min-width: 800px){
+
+  li {
+    font-size: 1vw;
+  }
+  h1 {
+    display: flex;
+    justify-content: center;
+    font-size: 5vw;
+  }
+
+  h2 {
+    display: flex;
+    font-size: 1.25vw;
+  }
+
+.main-nav ul {
```

Comment:
I don't see the difference between these rules and the ones at the top of the file. Why did these need to be in a media query?